source/common/router/config_impl.cc

```diff
@@ -450,7 +450,7 @@ bool VirtualHostImpl::usesRuntime() const {
 VirtualHostImpl::VirtualClusterEntry::VirtualClusterEntry(
     const envoy::api::v2::VirtualCluster& virtual_cluster) {
   if (virtual_cluster.method() != envoy::api::v2::RequestMethod::METHOD_UNSPECIFIED) {
-    method_ = envoy::api::v2::RequestMethod_Name(virtual_cluster.method());
+    method_.value(envoy::api::v2::RequestMethod_Name(virtual_cluster.method()));
```

Comment:
Yeah, I'm going to add the operator= (I already had it locally, but it seemed that this assignment was one-off and `.value()` was preferred way of working with `Optional`).

In any case, what should I do with existing `.value()` assignments? Convert them to `=` and remove `.value()` method or leave everything as-is?

Also, should I revert the `method_.value()` back to `method_ =`?